include/accelerator/rectifier.hpp

```diff
@@ -30,20 +28,32 @@ enum class MappingImpl {
     OpenCV
 };
 
+class RectifierBase {
+public:
+    virtual ~RectifierBase() {}
+    virtual Image::UniquePtr rectify(const Image &msg) = 0;
+    CameraInfo GetCameraInfoRect(void) {
+        // return copy of the current camera info rect
+        return CameraInfo(camera_info_rect_);
```

Comment:
I think copy constructor will be called automatically?
```suggestion
        return camera_info_rect_;
```